solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -205,6 +196,42 @@ contract LiquidityProtection is ILiquidityProtection, TokenHandler, Utils, Owned
         store.acceptOwnership();
     }
 
+    /**
+     * @dev transfers the ownership of the token holder
+     * can only be called by the contract owner
+     *
+     * @param _newOwner    the new owner of the token holder
+     */
+    function transferTokenHolderOwnership(address _newOwner) external ownerOnly {
+        tokenHolder.transferOwnership(_newOwner);
+    }
+
+    /**
+     * @dev accepts the ownership of the token holder
+     * can only be called by the contract owner
+     */
+    function acceptTokenHolderOwnership() external ownerOnly {
+        tokenHolder.acceptOwnership();
+    }
+
+    /**
+     * @dev transfer all equity from the store to the token holder
```

Comment:
migrates all funds from the store to the wallet

Reply:
Fixed.